lib/get.js

```diff
@@ -18,7 +18,7 @@ module.exports = function(bucket, options) {
   };
   if (options.logger) s3config.logger = options.logger;
   if (options.agent) s3config.httpOptions.agent = options.agent;
-  var s3 = options.s3 || new AWS.S3(s3config);
+  var s3 = options.s3 ? options.s3 : new AWS.S3(s3config);
```

Comment:
What does this fix?

Reply:
Its the same thing fixes nothing though, I can revert this change though

Reply:
If it doesn't change anything, I don't see any reason to include it.